# Notebook layout example

The three PDF files use this notebook with the default nbconvert layout, left prompts, and prompts above the code.

## Markdown

Text supports **bold**, *italic*, and `inline code`.

- First item
  - Nested item
- Second item

> A short quotation uses the template's block quote style.

## Code and output

The cells below use Python's standard library.

In [1]:
from statistics import mean

values = [12, 15, 18, 21]
print(f"Mean: {mean(values):.1f}")
[value ** 2 for value in values]

Mean: 16.5


[144, 225, 324, 441]

In [2]:
def describe(values):
    return {"min": min(values), "max": max(values), "count": len(values)}

describe(values)

{'min': 12, 'max': 21, 'count': 4}

## Mathematics and table

Inline mathematics: $a^2+b^2=c^2$.

$$\bar{x}=\frac{1}{n}\sum_{i=1}^{n}x_i$$

| Item | Value |
| --- | ---: |
| Minimum | 12 |
| Maximum | 21 |
| Mean | 16.5 |

## Long lines

Long code and output wrap within the page.

In [3]:
description = "This long line demonstrates how the template wraps code and saved output while keeping the original notebook source unchanged."
print(description)

This long line demonstrates how the template wraps code and saved output while keeping the original notebook source unchanged.
